BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -220,17 +220,6 @@ class BraintreeClient internal constructor(
         }
     }
 
-    /**
-     * @suppress
-     */
-    fun <T> isUrlSchemeDeclaredInAndroidManifest(urlScheme: String?, klass: Class<T>?): Boolean {
```

Comment:
This method was never used and was only covered by a failing integration test (also removed)